kegg_pathways_completeness/bin/plot_modules_graphs.py

```diff
@@ -67,8 +72,63 @@ def __init__(
         self.modules_completeness = modules_completeness
         self.graphs = graphs
         self.modules_definitions = modules_definitions
-        self.outdir = outdir
         self.modules_list = modules_list
+        # flags
+        self.use_pydot = use_pydot
+        # output directories
+        self.outdir = outdir
+        if not os.path.exists(self.outdir):
+            os.mkdir(self.outdir)
+        self.outdir_dot = os.path.join(self.outdir, 'dot')
+        if not os.path.exists(self.outdir_dot):
+            os.mkdir(self.outdir_dot)
+        self.outdir_png = os.path.join(self.outdir, 'png')
+        if not os.path.exists(self.outdir_png):
+            os.mkdir(self.outdir_png)
+
+    def create_graph_dot(self, name, presented, graph, pathways_schema):
+        # Create a pydot graph
+        dot = pydot.Dot(name=name, graph_type='digraph', comment=pathways_schema)
+
+        edges = graph[0].edges
+        max_weight = 0
+
+        for edge, count in zip(edges, range(len(edges))):
+            from_node = edge[0]
+            to_node = edge[1]
+            number = edge[2]
+
+            # Add nodes to the graph
+            dot.add_node(pydot.Node(str(from_node)))
+            dot.add_node(pydot.Node(str(to_node)))
+
+            # Extract edge attributes
+            label = edges._adjdict[from_node][to_node][number]['label']
+            weight = edges._adjdict[from_node][to_node][number]['weight']
+
+            if weight > 0:
+                if 1 / weight > max_weight:
+                    max_weight = int(1 / weight)
+
+            # Format weight string
```

Comment:
Hi @KateSakharova . Sorry I don't have a lot of context on this project. However, I can share some general code feedback. Please ignore if it does not apply due to context. 
I think this function could benefit from being made flatter and less nested. Also I think, we could do away with some of the conditionals. Here are some suggestions:

`
  
    def format_weight(weight):
        if weight in (0, 1):
            return str(weight)
        return f"1/{int(1/weight)}"
   
`

I think an explicitly named format_weight function, makes it a little easier to follow what's going on. 
Also In this approach, provides slightly less nesting of if statements.  This function would return the weight_string